server/app/views/questiontypes/ApplicantQuestionRendererImpl.java

```diff
@@ -36,10 +38,19 @@ protected enum InputFieldType {
 
   protected final ApplicantQuestion question;
   private final InputFieldType inputFieldType;
+  // HTML id tags for various elements within this question.
+  private final String id;
```

Comment:
Not blocking: I'd consider qualifying this as `questionId` or something along those lines.